aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/StateTransitionTests.kt

```diff
@@ -299,7 +299,13 @@ class StateTransitionTests : StateTransitionTestBase() {
         token = stateMachine.listen(
             {
                 val authState =
-                    it.takeIf { it is AuthState.Configured && it.authNState is AuthenticationState.SignedOut }
+                    it.takeIf {
+                        it is AuthState.Configured &&
+                            (
+                                it.authNState is AuthenticationState.SignedOut ||
+                                    it.authNState is AuthenticationState.Configured
```

Comment:
Configured is not final state, so AuthN will always be in either signedIn or signedOut state.